relion/protocols/protocol_subtract.py

```diff
@@ -102,41 +150,133 @@ def _defineParams(self, form):
                            "not at the corner.")
 
         line = form.addLine('Center coordinate (px)',
-                            condition='centerOnCoord',
+                            condition='centerOnCoord and relionInput',
                             help='Coordinate of the 3D center (in pixels).')
-        line.addParam('cX', IntParam, default=0, condition='centerOnCoord',
+        line.addParam('cX', IntParam, default=0, condition='centerOnCoord and relionInput',
                       label='X')
-        line.addParam('cY', IntParam, default=0, condition='centerOnCoord',
+        line.addParam('cY', IntParam, default=0, condition='centerOnCoord and relionInput',
                       label='Y')
-        line.addParam('cZ', IntParam, default=0, condition='centerOnCoord',
+        line.addParam('cZ', IntParam, default=0, condition='centerOnCoord and relionInput',
                       label='Z')
 
         form.addParam('newBoxSize', IntParam, default=-1,
+                      condition="relionInput",
                       label="New box size",
                       help="Provide a non-negative value to re-window the "
                            "subtracted particles in a smaller box size.")
 
+        form.addSection(label='CTF')
+        form.addParam('help', LabelParam,
```

Comment:
remove this param

Reply:
I guess this is the same problem described in the previous comment.